osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/config/MessagingConfig.java

```diff
@@ -24,8 +25,7 @@
  */
 @Configuration
 @PropertySource("classpath:osgp-adapter-protocol-dlms.properties")
-@PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true)
-@PropertySource(value = "file:${osgp/AdapterProtocolDlms/config}", ignoreResourceNotFound = true)
+@PropertySource(value = "file:/etc/osp/osgp-adapter-protocol-dlms.properties", ignoreResourceNotFound = true)
```

Comment:
Op zich mooi dat er nu 1 externe resource is, maar deze moet wel via een environment variable aangeleverd worden. Nu zit het pad hard in de app. Er moet een environment variabele geset worden in de context.xml op je systeem wanneer je deze gebruikt en die laten verwijzen naar /etc/osp/..